components/graph-line.vue

```diff
@@ -10,11 +10,14 @@
       v-if="collapsible"
       class="graph-line__toggle"
       label="Toggle"
-      @click="toggle"
+      @click="toggleCollapsedDataset(parameterId)"
```

Comment:
Why not toggleDataset?

Reply:
Because it is not that you turn on the dataset. It is about the collapsed state of the graph showing this dataset